experiments/plot_training.ipynb

```diff
@@ -11,28 +11,19 @@
     "import logging\n",
     "import os\n",
     "\n",
-    "import tensorflow as tf\n",
-    "\n",
     "from imitation.scripts.train_adversarial import train_ex\n",
     "os.chdir(\"../\")"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "scrolled": false
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
+    "# FIXME(sam): this is broken after SB3 port. Not touching b/c I'm not sure what it's meant to do.\n",
```

Comment:
No, it's the `.run()` call that is broken. I just put the comment at the top of the cell. The error doesn't look SB3/Torch-related, so I didn't bother debugging it:

![image](https://user-images.githubusercontent.com/1258993/87902392-56a18400-ca0e-11ea-9e10-1dc90a06f589.png)

Reply:
Missing the experiment demonstrations here. (Could download them using `experiments/download_experts.sh`)

I'm in favor of deleting this, no longer useful because we no longer have custom matplotlib plots.

Reply:
Now this has been deleted resolving this thread